src/coreclr/vm/jitinterface.h

```diff
@@ -713,7 +873,7 @@ class CEEJitInfo : public CEEInfo
         return m_fJumpStubOverflow;
     }
 
-    BOOL JitAgain()
+    virtual BOOL JitAgain() override
```

Comment:
nitpick: i think `virtual override` is redundant and we only need `override`?

Reply:
Right, will fix